README.md

```diff
@@ -12,9 +12,11 @@ Basically this plugin launches EsLint and collects its results into SonarQube. I
 
```

Comment:
Link to download the plugin on line #6 is wrong (forked repo.
Also we should update the readme explaining why we forked it, what are we doing different and if we have intention to make PRs to the original plugin.